### PR TITLE
docs: Get cozy-client-js intent doc

### PR DESCRIPTION
**Pull request by JF-Cozy**
Récupération de la doc des intents de cozy-client-js pour la mettre dans cozy-interapp

### CODE REVIEW

File: packages/cozy-interapp/README.md [hunk 1]
```diff
@@ -4,21 +4,114 @@ Library to create intents : actions that need to be fullfilled by another app th
...
-const intents = new Intents({ client })
-intents.create('io.cozy.apps', 'CREATE')
-  .start()
-  .then(intent => {
-    // do something with the result of the intent
-    // intent = {
-    //    id: '1337',
-    //    attributes: {...}
-    // }
+cozy.client.intents.create('EDIT', 'io.cozy.photos', {action: 'crop', width: 100, height: 100})
```
Comment (ptbrowne) [inline packages/cozy-interapp/README.md:28]: The `cozy.client.intents` is from another age where the intents were integrated in the client. The right way to instantiate the `Intents` is the one that has been removed here:

```
const client = new CozyClient({ uri, token })
const intents = new Intents({ client })
intents.create('io.cozy.apps', 'CREATE')
```

I think it is sufficient to replace

```patch
- cozy.client.intents.create('EDIT', 'io.cozy.photos', {action: 'crop', width: 100, height: 100})
+ const client = new CozyClient({ uri, token })
+ const intents = new Intents({ client })
+ intents.create('io.cozy.apps', 'CREATE')
```

The EDIT intent for io.cozy.photos does not exist so let's not put it in the doc :)

Comment (JF-Cozy) [reply packages/cozy-interapp/README.md]: Ok :) Cela semble être plutôt `intents.create( 'CREATE', 'io.cozy.apps')` (cf https://github.com/cozy/cozy-libs/blob/doc/intent/packages/cozy-interapp/src/intents.js#L12 et https://github.com/cozy/cozy-ui/blob/feat/Intents/react/IntentIframe/IntentIframe.jsx#L39)

**Review by ptbrowne — APPROVED** [review]

File: packages/cozy-interapp/README.md [hunk 2]
```diff
@@ -22,3 +39,87 @@ intents.create('io.cozy.apps', 'CREATE')
     // }
   })
 ```
+
+Example to use `removeIntentFrame()` method (by passing the flag `exposeIntentFrameRemoval` flag):
+```js
```
Comment (ptbrowne) [inline packages/cozy-interapp/README.md:44]: Line break is missing before code block, GitHub supports this but I am not sure that mkdocs will like it.

**Review by ptbrowne — APPROVED** [review]
Thanks, please add line breaks before code blocks. We need to add a lint for this, it is too easy to miss.

https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/cozy/cozy-ui.*%24+remark&patternType=literal

**Comment by ptbrowne** [comment]
For the linter: https://github.com/cozy/cozy-libs/pull/1243 I'll rebase when this PR is merged

**Pull request merged**